kodilla-testing/src/test/java/com/kodilla/testing/forum/statistics/CalculateStatisticsTestSuite.java

```diff
@@ -0,0 +1,35 @@
+package com.kodilla.testing.forum.statistics;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class CalculateStatisticsTestSuite {
+    @Test
+    public void testCalculateAdvStatistics(){
+        Statistics statisticsMock = mock(Statistics.class);
```

Comment:
you test only one case, and forget about rest. Your test coverage is not enough. And you have bugs in your code (if you had written comprehensive tests, you would have found them :) )